uktrade/data-hub-frontend
# Deal with delete reminder pagination issue

## Description of change

Deals with potential issues around pagination and deleting reminders by bringing in the next item each time a reminder is deleted. The old item remains in place with a message to say it has been deleted and the next item is brough in from the top of the next page.

## Test instructions

- Create more than 10 reminders through django admin
- Try deleting some and see that the next item is brought in - when there are fewer than 11 the pagination should no longer be shown


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
